src/src/AdminBundle/Resources/views/EventType/index.html.twig

```diff
@@ -0,0 +1,40 @@
+{% extends "@AdminBundle/Resources/views/EventType/base.html.twig" %}
+
+{% block content %}
+    <h1 class="contents">
```

Comment:
```
<h1 class="contents">
         <span>Event Type</span>
         <a class='btn btn-primary float-sm-right' href="{{ path('event_type_add') }}">Add new Event Type</a>
 </h1>
```

takyto kod mame asi na 4 miestach, nespravime z toho nieco co sa bude dat includnut? (nebolo by to sucastou tejto issue)

Reply:
Bolo by to nice, ale robila by som to az uplne nakoniec ako nice to have veci. Radsej venujme cas rozbehaniu posielania emailov.